build/yamls/base/conf/antrea-agent.conf

```diff
@@ -102,15 +102,22 @@ featureGates:
 # L4 transport protocols.
 #flowCollectorAddr: "flow-aggregator.flow-aggregator.svc:4739:tcp"
 
-# Provide flow poll interval as a duration string. This determines how often the flow exporter dumps connections from the conntrack module.
-# Flow poll interval should be greater than or equal to 1s (one second).
+# Provide flow poll interval as a duration string. This determines how often the
+# flow exporter dumps connections from the conntrack module. Flow poll interval
+# should be greater than or equal to 1s (one second).
 # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 #flowPollInterval: "5s"
 
-# Provide flow export frequency, which is the number of poll cycles elapsed before flow exporter exports flow records to
-# the flow collector.
-# Flow export frequency should be greater than or equal to 1.
-#flowExportFrequency: 12
+# Provide active flow export timeout, which is a timeout for each active flow to
+# send flow records to the collector with respect to its last export time.
```

Comment:
> Provide the active flow export timeout, which is the timeout after which a flow record is sent to the collector for active flows. Thus, for flows with a continuous stream of packets, a flow record will be exported to the collector once the elapsed time since the last export event is equal to the value of this timeout.